echo_api.rb

```diff
@@ -8,6 +8,13 @@
 require 'base64'
 require 'rack/cors'
 
+if ENV['OPENTRACING_TRACER'] == "jaeger"
```

Comment:
you can just move all the requires to the block below where you configure it

Reply:
👍 Fixing it.